command/helper/config.go

```diff
@@ -72,8 +72,8 @@ const defaultBlockTime uint64 = 2
 // DefaultConfig returns the default server configuration
 func DefaultConfig() *Config {
 	return &Config{
-		Chain:          "test",
-		DataDir:        "./test-chain",
+		Chain:          "polygon-edge",
+		DataDir:        "./polygon-edge-chain",
```

Comment:
You should include this in the .gitignore file as well

Reply:
Added to .gitignore in 189841266f8a705f8ae512daceccb5556794089f